tests/integration/__tests__/cache.js

```diff
@@ -227,6 +227,95 @@ describe("--cache option", () => {
       );
     });
 
+    it("re-formats after execution without write.", async () => {
```

Comment:
I don't think it's a good idea to relay on another `it` call, should format twice in one block. otherwise it.only will fail.

Reply:
Ah, seems already did. Sorry.